HW_4_6_Chernysh/HW_4_6_Chernysh/Models/Song.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HW_4_6_Chernysh.Models
+{
+    public class Song
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public TimeSpan Duration { get; set; }
+        public DateTime ReleasedDate { get; set; }
+        public virtual int GenreId { get; set; }
```

Comment:
1. It's not needed virtual keyword here. It's not navigation property.
2. GenreId should be nullable